src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabServerPullRequestDecorator.java

```diff
@@ -155,10 +158,10 @@ public void decorateQualityGateStatus(AnalysisDetails analysis, AlmSettingDto al
                 }
             }
 
-            String coverageValue = analysis.findQualityGateCondition(CoreMetrics.NEW_COVERAGE_KEY)
+            String coverageValue = analysis.findQualityGateCondition(CoreMetrics.COVERAGE_KEY)
                     .filter(condition -> condition.getStatus() != QualityGate.EvaluationStatus.NO_VALUE)
                     .map(QualityGate.Condition::getValue)
-                    .orElse("0");
+                    .orElse(NO_COVERAGE_VALUE);
```

Comment:
Why are we setting it to a value, rather than `null` or potentially just remaining as an optional with a subsequent `hasValue` check?

Reply:
Good question. My Java is a little bit rusty so I went for this option, but I'm happy to take suggestions here.

Reply:
I'll check where the variable is referenced and get you a suggestion shortly

Reply:
Still waiting on feedback for this one.